### ISSUE TITLE
Training 3 classes on pretrained model with yolov3_voc2012.pth as checkpoint leads to dimensional problems (obviously!)

### ISSUE THREAD

**Issue by mth1996** [johnnylord/trytry-detection#5]
Hey johnnylord,

i would really like to benefit from your pretrained model. Because of that i tried downloading the yolov3_voc2012.pth file and imported it as checkpoint to the algorithm. The problem is, that that leads to dimensional problems, because my model runs with 3 classes. I think its because of the scaled predictions.

Is there an possibility to take profit of your pretrained weights to train the model after with less classes? 

Greets!

**Comment by johnnylord** [comment]
Well, I probably provide customized training instruction in the near future ...

To take the advantage of my pretrained weights, you need to load the state_dict in less strict mode, and modify the model architecture of the prediction head.

I haven't test it yet.
